apps/backend/src/modules/auth/auth.service.ts

```diff
@@ -1,14 +1,50 @@
+import { ErrorMessage } from "~/libs/enums/enums.js";
+import { Encrypt } from "~/libs/modules/encrypt/encrypt.js";
+import { HTTPCode } from "~/libs/modules/http/http.js";
+import { AuthError } from "~/modules/auth/auth.js";
```

Comment:
This will create a circular dependency. Let's import from `~/modules/auth/libs/types/types.js`

Reply:
I moved AuthError to a separate directory under exceptions since it’s not a type and shouldn't be exported from the types file. If this approach is incorrect, please let me know and I will adjust it.